helpers/calendar/analytics_helper.py

```diff
@@ -94,48 +94,57 @@ def get_calendar_id_name(self, query):
+                                 event_start_time,
+                                 event_end_time):
         """ Get all events in a room
          :params
-            - calendar_id - for specific room
-            - min_limit, max_limit(Time range)
+            - room_id - for specific room
+            - event_start_time, event_end_time(Time range)
         """
-        try:
-            events_result = get_google_calendar_events(
-                calendarId=calendar_id, timeMin=min_limit,
-                timeMax=max_limit, singleEvents=True,
-                orderBy='startTime')
-        except Exception:
-            raise GraphQLError("Resource not found")
-        all_events = events_result.get('items', [])
-        return all_events
+        events = EventsModel.query.all()
+        result = []
+        for event in events:
+            if event.room_id == room_id:
+                room_event = {
+                    'room_id': event.room_id,
+                    'event_start_time': event.start_time,
+                    'event_end_time': event.end_time,
+                    'event_title': event.event_title,
+                    'participants': event.number_of_participants,
+                    'event_id': event.event_id,
+                }
+                result.append(room_event)
+            if event.room_id == room_id and \
+                (event.start_time < event_start_time or
+                    event.end_time > event_end_time):
+                result.remove(room_event)
+        return result
 
-    def get_event_details(self, event, calendar_id):
+    def get_event_details(self, query, event, room_id):
         """ Filter details of an event
          :params
             - event
         """
         event_details = {}
-        if event.get('attendees'):
-            for resource in event.get('attendees'):
-                if resource.get('resource') and resource.get('email') == calendar_id:  # noqa: E501
-                    event_details["minutes"] = CommonAnalytics.get_time_duration_for_event(  # noqa: E501
-                        self,
-                        event['start'].get('dateTime', event['start'].get('date')),  # noqa: E501
-                        event['end'].get('dateTime', event['end'].get('date'))
-                    )
-                    event_details["roomName"] = resource.get(
-                        'displayName') or None
-                    event_details["summary"] = event.get("summary")
-        elif event.get('organizer') and event.get(
-                'organizer').get('email') == calendar_id:
-            event_details["roomName"] = event.get(
-                'organizer').get('displayName') or None
-            event_details["summary"] = event.get("summary")
+        rooms_available = CommonAnalytics.get_room_details(
+            self, query)
+        for room in rooms_available:
+            event_details["minutes"] = CommonAnalytics.get_time_duration_for_event(  # noqa: E501
```

Comment:
Remove noqa within the parentheses

Reply:
Hello @Bonifase, the noqa is there because the line is too long and I couldn't break it, (flake8 issues) If there is a way of breaking it, kindly advise.

Reply:
put it outside the brackets

Reply:
This has been worked on @Bonifase